packages/server/src/auth/newproject.ts

```diff
@@ -75,6 +75,7 @@ export async function createProject(
     resourceType: 'Project',
     name: projectName,
     owner: createReference(user),
+    strictMode: true,
```

Comment:
@medplum/dev I want to call attention to this line in particular, which makes strict mode the default for all new projects.